lib/src/ui/pages/authentication/login/login_notifier.dart

```diff
@@ -25,23 +26,22 @@ class LoginNotifier extends ChangeNotifier {
   VoidCallback returnToProfile;
 
   void validateFieldsAndSignIn() {
+    _setNotifierState(_state.copyWithLoading(isLoading: true));
+
     _validateFields(
       onInvalid: () => notifyListeners(),
       onValid: () => _signIn(),
     );
   }
 
   Future<void> _signIn() async {
-    _setNotifierState(_state.copyWithLoading(isLoading: true));
-
     try {
       await _authenticationRepository.signIn(_state.email, _state.password);
-      _setNotifierState(_state.copyWithLoading(isLoading: false));
+      _setNotifierState(const LoginState());
       returnToProfile?.call();
     } on AuthenticationApiException catch (exception) {
-      _setNotifierState(_state.copyWithLoadingAndAuthError(
-        authenticationError: _authenticationMessageProvider.getMessage(exception),
-        isLoading: false,
+      _setNotifierState(LoginState(
```

Comment:
czyścisz stan UI po requeście, jakbym dostał jakiś błąd połączenia albo nawet podał złe hasło to nie chciałbym żeby mi wyczyścił input